### PR TITLE
displaying a closed discussion and disabled reply

### PR DESCRIPTION
**Pull request by vinyll**

### CODE REVIEW

File: udata/templates/discussion/item.html [hunk 1]
```diff
@@ -37,6 +38,7 @@ <h4 class="list-group-item-heading">
         {{ _('Add a comment') }}
     </h4>
 </a>
+{% endif %}
```
Comment (davidbgk) [inline udata/templates/discussion/item.html]: IIRC, you might even close the `if` after the next `div`?

**Comment by davidbgk** [comment]
r+wc

File: udata/templates/discussion/item.html (hunk 1 above)
Comment (vinyll) [inline udata/templates/discussion/item.html]: I may misunderstand here but commenting the next `<div>` would remove the link, user avatar and new discussion form.
We only want to remove the _Add a comment_ as the discussion is closed yet the threat is still open.

Comment (davidbgk) [inline udata/templates/discussion/item.html]: My bad, you're right!

**Pull request merged**